gui/RemoveItem.java

```diff
@@ -216,33 +243,42 @@ public void setup() {
 		// The students on the Roster List for this class
 		ArrayList<ListEntry> rlist = new ArrayList<ListEntry>();
 
-		for (String s : AdminUI.roster) {
+		for (String s : MentorUI.roster) {
```

Comment:
this will probably change but its ok for now. there will probably be a roster manager to deal with multiple lists